Assets/Scripts/Game/BuildingDirectory.cs

```diff
@@ -157,11 +155,31 @@ public List<BuildingSummary> GetBuildingsOfFaction(int factionId)
 
         public List<BuildingSummary> GetHousesForSale()
         {
-            // TODO: add other random houses.
+            int maxForSale = Mathf.Min(buildingDict.Count / 10, 20);
             List<BuildingSummary> forSale = new List<BuildingSummary>();
+            List<BuildingSummary> candidates = new List<BuildingSummary>();
             foreach (BuildingSummary building in buildingDict.Values)
+            {
                 if (building.BuildingType == DFLocation.BuildingTypes.HouseForSale)
+                {
                     forSale.Add(building);
+                }
+                else if (building.BuildingType > DFLocation.BuildingTypes.House1 &&
+                         building.BuildingType < DFLocation.BuildingTypes.House4 &&
```

Comment:
Should be <= to include House4 type

Reply:
Good catch. Copied from an exclusion conditional and brain farted that switch around.